plugins/sentinelone/komand_sentinelone/util/api.py

```diff
@@ -78,7 +122,7 @@ def __get_searches(agent_details: str) -> list:
 
     @staticmethod
     def clean_results(results):
-        return loads(dumps(results).replace("null", '"None"'))
+        return clean(loads(dumps(results).replace("null", '"None"')))
```

Comment:
Why single and double quotes on `None` here?

Reply:
Here we take `dumps` object to string. This change empty values to null. On the json side it looks like: `"key": null`. We replace this null value with: `"key": "None"`, so in clean we can check this value: `value == "None"` and in this situations we return key with empty array instead of null or none.